pkg/apis/crd/v1alpha1/types.go

```diff
@@ -51,6 +52,8 @@ const (
 	// ActionForwardedOutOfOverlay indicates that the packet has been forwarded out of the network
 	// managed by Antrea. This indicates that the Traceflow request can be considered complete.
 	ActionForwardedOutOfOverlay TraceflowAction = "ForwardedOutOfOverlay"
+	ActionMarkedForSNATByHost   TraceflowAction = "MarkedForSNATByHost"
```

Comment:
Question - is this for SNAT on Egress Node only? If so, should we name it something like `ActionSNATOnEgressNode`?

Reply:
Or maybe just `ActionMarkedForSNAT"?

Reply:
> Question - is this for SNAT on Egress Node only? If so, should we name it something like `ActionSNATOnEgressNode`?

yes